### PR TITLE
Update navtabs component in lexicon-base

### PR DESCRIPTION
**Pull request by dustinryerson**

### CODE REVIEW

**Comment by dustinryerson** [comment]
Forgot, need to update the new migration page. I'll update this pull with those changes.

File: src/content/nav_tabs.html [hunk 1]
```diff
@@ -236,45 +240,45 @@
...
-					<li><a href="#1">Supported Clients</a></li>
-					<li><a href="#1">Permissions</a></li>
-					<li><a href="#1">Sharing</a></li>
-					<li class="divider"></li>
-					<li><a href="#1">Scope</a></li>
-				</ul>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link active" aria-controls="fields" href="#fields04" data-toggle="tab" role="tab">Fields</a></li>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link" aria-controls="settings" href="#settings04" data-toggle="tab" role="tab">Settings</a></li>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link disabled" href="#documents">Documents</a></li>
+			<li class="dropdown nav-item col-xs-3 row" role="presentation">
```
Comment (dustinryerson) [inline src/content/nav_tabs.html]: For some reason the ```Configuration``` tab would not align with the other tabs unless I added the ```row``` class. There may be a better way to fix this.

File: src/scss/lexicon-base/_nav-tabs.scss [hunk 2]
```diff
@@ -34,7 +34,7 @@
 	}
 }
 
-@media screen and (max-width: $grid-float-breakpoint-max) {
```
Comment (dustinryerson) [inline src/scss/lexicon-base/_nav-tabs.scss]: I couldn't find a replacement for the ```$grid-float-breakpoint-max``` variable. Is there a better option to use that I can use for future component updates?

**Comment by dustinryerson** [comment]
This is ready for review 👍

**Comment by pat270** [comment]
Hey @dustinryerson, we shouldn't add the row class on the configuration tab. The reason for the overflow is due to the .nav-tabs .nav-item + .nav-item { margin-left: 0.2rem; } from bootstrap. We can set it to 0 for .nav-tabs .nav-item + .nav-item[class*='col-'] {}. 

We should also change the selectors from nav-tabs > li and nav-tabs > li > a to .nav-tabs .nav-item and .nav-tabs .nav-link

Nav Tabs Default should remain and be styled like Nav Tabs Default at http://liferay.github.io/lexicon/content/nav-tabs/#nav-tabs-default.

Let me know if you have any questions.

**Comment by dustinryerson** [comment]
Hey @pat270, I've made the changes you detailed above. Please let me know if I missed anything.

Thanks!

**Comment by pat270** [comment]
Merged, Thanks!

View my changes: https://github.com/pat270/lexicon/compare/5d401626e94de2ee99d1c431191996e5a19eee9c...1fa510925658faf1f54646661d5b6ee7e028435c
View total diff:  https://github.com/pat270/lexicon/compare/b64d435706a155c495d3fc8a4f008fd976410126...1fa510925658faf1f54646661d5b6ee7e028435c

**Pull request closed without merging**